src/order-and-subscriptions/OrderAndSubscriptionsPage.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import Subscriptions from '../subscriptions';
+import Payments from '../order-history';
```

Comment:
Suggestion - could be less confusing if this was imported as `Orders`

Reply:
Okay 👍